docs/website/docs/general-usage/schema-contracts.md

```diff
@@ -129,27 +130,24 @@ except PipelineStepFailed as pip_ex:
   if pip_ex.step == "extract":
     if isinstance(pip_ex.__context__, DataValidationError):
       ...
-
-
 ```
 
 `DataValidationError` provides the following context:
-1. `schema_name`, `table_name` and `column_name` provide the logical "location" at which the contract was violated.
-2. `schema_entity` and `contract_mode` tell which contract was violated
-3. `table_schema` contains the schema against which the contract was validated. May be Pydantic model or `dlt` TTableSchema instance
-4. `schema_contract` the full, expanded schema contract
-5. `data_item` causing data item (Python dict, arrow table, pydantic model or list of there of)
-
+1. `schema_name`, `table_name`, and `column_name` provide the logical "location" at which the contract was violated.
+2. `schema_entity` and `contract_mode` indicate which contract was violated.
+3. `table_schema` contains the schema against which the contract was validated. It may be a Pydantic model or a `dlt` TTableSchema instance.
```

Comment:
```suggestion
3. `table_schema` contains the schema against which the contract was validated. It may be a Pydantic model or a dlt `TTableSchema` instance.
```